src/utils/claim.ts

```diff
@@ -0,0 +1,70 @@
+import { Wallet } from "ethers";
+import { Multicall3__factory, RewardsDistributor__factory } from "@morpho-labs/morpho-ethers-contract";
+import addresses from "@morpho-labs/morpho-ethers-contract/lib/addresses";
+import { StorageService } from "./StorageService";
+import { formatUnits } from "ethers/lib/utils";
+
+/**
+ * Claim rewards for multiple users using an EOA on Mainnet.
+ * @notice Addresses are hardcoded to improve user address checking. (really encouraged)
+ *
+ * @param userAddresses A  list of user addresses to claim rewards for
+ * @param storageService A storage service to read the proofs from
+ * @param signer The signer to use to send the transaction
+ */
+export const claim = async (userAddresses: string[], storageService: StorageService, signer: Wallet) => {
+  if ((await signer.provider.getNetwork()).chainId !== 1)
+    throw new Error("Claiming rewards is only supported on mainnet");
+
+  const rewardsDistributor = RewardsDistributor__factory.connect("0x3b14e5c73e0a56d607a8688098326fd4b4292135", signer);
```

Comment:
Would be better to isolate the address into a proper constant

Reply:
You can check what Ive written in the jsdoc of the function, its intentional.